### PR TITLE
Indicate blockers

### LINKED ISSUE #19: Show blocker bugs
When a bug is blocked by another bug, that should be shown in the UI.

### PR DESCRIPTION
**Pull request by willkg**
Fixes #19 

This is slightly different than my original plan. I decided that "loading blockers" was a step of "load bugs", so I made it more like "load comments".

I tweaked the CSS for the needsinfo stuff so it looks a little bit better when there are more than one.

r?

### CODE REVIEW

File: edwin/client/static/js/actions/TimelineActions.js [hunk 1]
```diff
@@ -115,9 +121,36 @@ export function loadCommentTags(bugIds) {
...
+    return bzAPI.getBugs(bugQuery)
+    .then(newBugs => {
+      Dispatcher.dispatch({
+        type: TimelineConstants.ActionTypes.SET_BLOCKER_BUGS,
+        newBugs,
+      });
+    })
+    .then(() => ProgressActions.endTask('Load blockers'));
+  } else {
+    return undefined;
```
Comment (mythmon) [inline edwin/client/static/js/actions/TimelineActions.js]: Uh oh. It probably isn't good to sometimes return a Promise and sometimes not. Maybe return something like `Promise.resolve()` (return a promise that is resolved to `undefined`) instead?

File: edwin/client/static/js/stores/BugStore.js [hunk 2]
```diff
@@ -211,6 +235,20 @@ BugStore.dispatchToken = Dispatcher.register((action) => {
       BugStore.emitChange();
       break;
 
+    case ActionTypes.SET_BLOCKER_BUGS:
+      // Merge newBugs into bugMap
+      for (let bug of action.newBugs) {
+        bugMap = bugMap.update(bug.id, new Immutable.Map(), oldBug => {
+          let newBug = Immutable.fromJS(bug);
+          return oldBug.merge(newBug);
+        });
```
Comment (mythmon) [inline edwin/client/static/js/stores/BugStore.js]: I think these 4 lines could be replaced with `bugMap = bugMap.mergeIn([bug.id], Immutable.fromJS(bug))`;

I think.

**Comment by mythmon** [comment]
That's... hideous. But it works, so that's cool.

This will bring bugs that aren't tagged with `edwin-${team}` into the bug list as well, right, if they are blocking bugs already in the list? I think that is probably fine.

**Comment by willkg** [comment]
It does add non-team bugs into the bug map. So we should probably have the bug lists that bugMap exposes filter them out. I can look into that.

I'll make the other fixes today, too.

**Comment by willkg** [comment]
^^^ Fixes the issues you found, plus it fixes issue #61.

**Comment by mythmon** [comment]
That should work. Lets go with it.

**Pull request merged**